lib/wire_structs.ml

```diff
@@ -45,6 +45,30 @@ module Ipv4_wire = struct
     | `UDP    -> 17
 end
 
+module Arpv4_wire = struct
+
+  cstruct arp {
+    uint8_t dst[6];
+    uint8_t src[6];
+    uint16_t ethertype;
+    uint16_t htype;
+    uint16_t ptype;
+    uint8_t hlen;
+    uint8_t plen;
+    uint16_t op;
+    uint8_t sha[6];
+    uint32_t spa;
+    uint8_t tha[6];
+    uint32_t tpa
+  } as big_endian
+
+  cenum op {
+    Op_request = 1;
+    Op_reply
+    } as uint16_t
+
+end
```

Comment:
I'm in favour of breaking the consistency of the library and put ARP wire format directly in `arp.ml` (or in `arp_wire.ml`).